### ISSUE TITLE
Create a list of algorithms and techniques to be used

### ISSUE THREAD

**Issue by krishnaths** [krishnaths/training_ground#21]

**Comment by krishnaths** [comment]
Datasets available - iris




Functions:
train_test_split -  Splitting of dataset using 
metrics.accuracy_score - to find the accuracy of the score.
metrics.confusion_matrix - to find the confusion of classification problems
Pipeline can be used to create a pipeline by supplying a scalar and model.







Models:
SGDClassifier - Linear classifier, Stochastic Gradient Descent

**Comment by krishnaths** [comment]
Bias - Variance : Can cause underfitting and overfitting.
